gen/_template/security.tmpl

```diff
@@ -29,6 +29,18 @@ func findAuthorization(h http.Header, prefix string) (string, bool) {
 }
 
 {{ range $s := $.Securities }}
+{{- if $s.Format.IsOAuth2Security }}
+var oauth2Scopes = map[string][]string {
+{{- range $operationName, $scopes := $s.Scopes }}
+  {{ quote $operationName }}: []string{
+    {{- range $scope := $scopes }}
+      {{ quote $scope }},
+    {{- end}}
+  },
+{{- end }}
+}
```

Comment:
According to our [.editorconfig](https://github.com/ogen-go/ogen/blob/main/.editorconfig#L11-L13) we use tabs in template files.

Reply:
Changed :+1: